### ISSUE TITLE
Structure des fichiers - Parkings

### ISSUE THREAD

**Issue by TuThoThai** [etalab/transport-profil-netex-fr#119]
Entrée ZIP: `PARKING.xml`

**Travail en cours**

**Comment by TuThoThai** [comment]
Note : travail à faire en lien avec le GT9

**Comment by TuThoThai** [comment]
Pour le moment status quo avec l'ensemble des éléments cités dans cette partie du profil, à l'exception des éléments d'offre tarifaire attendus dans le fichier `fare.xml`

Les éléments restants sont : 
- PARKING (contiendra par inclusion tous les autres objets comme PARKING AREAs, VEHICLE ENTRANCEs, etc.)
- GENERAL ORGANISATION
- BRANDING
- VEHICLE TYPE

**Comment by prhod** [comment]
A confirmer : A part Parking et les objets inclus, les autres objets semblent communs avec les autres sous-parties du profil france, donc a ajouter dans Elements communs (resources.xml)

**Comment by TuThoThai** [comment]
Après discussion, corrections faites par @prhod pour harmoniser les objets inclus avec les autres parties du profil France. Voir commit 6d02c7cc27d63c8d3f8bbd8690ab2e4a3b8d0d7e

**Comment by nlehuby** [comment]
Est-ce qu'on peut ajouter explicitement `ParkingBay` (places de stationnement individuelles) dans les éléments possibles ?

Sinon, je comprends que si on veut juste exporter des places PMR en voirie, on est obligés de créer un faux Parking qui n'a pas vraiment d'existence réelle sur le terrain.

**Comment by TuThoThai** [comment]
> Est-ce qu'on peut ajouter explicitement `ParkingBay` (places de stationnement individuelles) dans les éléments possibles ?
> 
> Sinon, je comprends que si on veut juste exporter des places PMR en voirie, on est obligés de créer un faux Parking qui n'a pas vraiment d'existence réelle sur le terrain.

Oui, effectivement ! Rajouté dans le commit aaadd48c1fa58945e8952fb70b930dc5f04fe376

**Comment by TuThoThai** [comment]
PR mergée, je clos le ticket.
